lib/Dialect/FIRRTL/Translation/FIRParser.cpp

```diff
@@ -3,8 +3,8 @@
 // This implements a .fir file parser.
 //
 //===----------------------------------------------------------------------===//
```

Comment:
Instead of `Translation`, shouldn't the directory be lib/Dialect/FIRRTL/Import/FIRParser.cpp  to match the library name?

Reply:
Sure, I'll change it to match.